velox/functions/sparksql/aggregates/BloomFilterAggAggregate.cpp

```diff
@@ -100,10 +100,10 @@ class BloomFilterAggAggregate : public exec::Aggregate {
       bool /*mayPushdown*/) override {
     decodeArguments(rows, args);
     computeCapacity();
-    auto mayHaveNulls = decodedRaw_.mayHaveNulls();
+
     rows.applyToSelected([&](vector_size_t row) {
-      if (mayHaveNulls) {
-        checkBloomFilterNotNull(decodedRaw_, row);
+      if (decodedRaw_.isNullAt(row)) {
```

Comment:
Check the `mayHaveNulls` first, and then check if this row is null.
Most cases `mayHaveNulls` is false, it will be more efficient.